seatunnel-e2e/seatunnel-connector-v2-e2e/connector-http-e2e/src/test/java/org/apache/seatunnel/e2e/connector/http/HttpIT.java

```diff
@@ -357,6 +357,9 @@ public void testSourceToAssertSink(TestContainer container)
         Container.ExecResult execResult19 =
                 container.executeJob("/http_page_increase_start_num.conf");
         Assertions.assertEquals(0, execResult19.getExitCode());
+
+        Container.ExecResult execResult21 = container.executeJob("/http_to_console.conf");
+        Assertions.assertEquals(0, execResult21.getExitCode());
```

Comment:
I perfer use unit test to check it rather than e2e. Unit test can check all return value more easier. Current e2e test case you added even not check result data.